selfdrive/locationd/paramsd.py

```diff
@@ -213,7 +213,7 @@ def migrate_cached_vehicle_params_if_needed(params_reader: Params):
     last_parameters_msg.liveParameters.stiffnessFactor = last_parameters_dict['stiffnessFactor']
     last_parameters_msg.liveParameters.angleOffsetAverageDeg = last_parameters_dict['angleOffsetAverageDeg']
     params_reader.put("LiveParameters", last_parameters_msg.to_bytes())
-  except (json.JSONDecodeError, KeyError):
+  except Exception:
```

Comment:
like what? can we be more explicit? and add a cloudlog.warn or .error here

Reply:
its expected for this to silently fail if anything prevents it from migrating the cached params (like invalid format, missing keys, invalid types, decoding errors etc...)